examples/solvers/pdhg_denoising_ROF_primal_acceleration.py

```diff
@@ -0,0 +1,81 @@
+"""Total variation denoising using the primal-dual hybrid gradient algorithm.
+
+Solves the optimization problem
+
+    min_{x >= 0}  1/2 ||x - g||_2^2 + lam || |grad(x)| ||_1
+
+Where ``grad`` the spatial gradient and ``g`` is given noisy data.
+
+For further details and a description of the solution method used, see
+:ref:`chambolle_pock` in the ODL documentation.
+"""
+
+import numpy as np
+import scipy
+import odl
+
+# Read test image: use only every second pixel, convert integer to float,
+# and rotate to get the image upright
+image = np.rot90(scipy.misc.ascent()[::2, ::2], 3).astype('float')
+shape = image.shape
+
+# Rescale max to 1
+image /= image.max()
+
+# Discretized spaces
+space = odl.uniform_discr([0, 0], shape, shape)
+
+# Original image
+orig = space.element(image)
+
+# Add noise
+image += np.random.normal(0, 0.1, shape)
+
+# Data of noisy image
+noisy = space.element(image)
+
+# Gradient operator
+gradient = odl.Gradient(space, method='forward')
+
+# Matrix of operators
+op = gradient
+
+# Set up the functionals
+
+# Isotropic TV-regularization: l1-norm of grad(x)
+reg_param = 0.15
+l1_norm = reg_param * odl.solvers.L1Norm(gradient.range)
+
+# Make separable sum of functionals, order must correspond to the operator K
+f = l1_norm
+
+# Data fit with non-negativity constraint
```

Comment:
This file is removed and the primal acceleration bit is on the algorithmic comparison